src/Filters/ConvertPsuedoHeadings.php

```diff
@@ -20,14 +20,26 @@ public function apply(string $html): string
             $html
         );
 
+        $html = preg_replace(
+            '/^(<strong>(.{0,80}?)<\/strong>(?:\s+)?(<br *\/?>)?(?:\s+)?)+/si',
```

Comment:
I see that this is from before but why only 80 characters? Seems like an odd boundary!

Reply:
seems like `(.{0,80}?)` can be replaced by `(.*?)` without any issues as it is non greedy

Reply:
I guess we wanted to impose a limit to the heading length on jobs coming in from Autoposter. I agree it looks somewhat arbitrary, but felt it was out of scope of the story to update them

Reply:
Yeah definitely out of scope